ts/Extensions/Boost/BoostSeries.ts

```diff
@@ -235,6 +236,27 @@ function compose<T extends typeof Series>(
         ).forEach((method): void =>
             wrapSeriesFunctions(seriesProto, seriesTypes, method)
         );
+        wrap(
+            PointClass.prototype,
+            'firePointEvent',
+            function (
+                this: typeof PointClass.prototype,
+                proceed, type, e
+            ): boolean | undefined {
+                if (type === 'click' && this.series.boosted) {
+                    const point = e.point,
+                        { xAxis, yAxis } = point.series,
+                        distance = Math.sqrt(
+                            Math.pow(point.plotX + xAxis.pos - e.chartX, 2) +
+                            Math.pow(point.plotY + yAxis.pos - e.chartY, 2)
+                        );
+
+                    if (distance > 10) {
```

Comment:
How about `>= point.series.options.marker?.radius ?? 10`? For `scatter` it makes sense. Falls back to `10` for e.g. `column`. Not sure if similar check can be done for `bubble` - at a first glance, the point's z-based marker pixel size computation is done only in the boost scope for drawing, not assigned to the point or data.